Library/Homebrew/sandbox.rb

```diff
@@ -141,8 +141,15 @@ def path_filter(path, type)
   class SandboxProfile
     SEATBELT_ERB = <<-EOS.undent
       (version 1)
+      (deny network-inbound)
+      (deny network-outbound)
       (debug deny) ; log all denied operations to /var/log/system.log
       <%= rules.join("\n") %>
+      (allow network-outbound
+        (subpath "/tmp")
+        (subpath "/private")
+        (subpath "#{HOMEBREW_TEMP}")
```

Comment:
> This path is needed for the socket we use to communicate with build.rb.

According to [API guidance](https://reverse.put.as/wp-content/uploads/2011/09/Apple-Sandbox-Guide-v1.0.pdf). It should be `(remote unix|unix-socket [path-literal PATH])`.

Also FYI, the source downloading happens inside sandbox.